packages/core/src/components/modal/modal.tsx

```diff
@@ -79,6 +79,12 @@ export class Modal implements ComponentInterface {
       }
 
       this.htmlDialogElement.showModal();
+
+      this.htmlDialogElement.addEventListener("keydown", (e: KeyboardEvent) => {
```

Comment:
Heb je gekeken naar de `close` en `cancel` events van `dialog` ? Kunnen we daar iets mee?

- https://developer.mozilla.org/en-US/docs/Web/API/HTMLDialogElement/cancel_event
- https://developer.mozilla.org/en-US/docs/Web/API/HTMLDialogElement/close_event

Reply:
Zie https://github.com/dso-toolkit/dso-toolkit/pull/3167#discussion_r2129132387 voor het vervolg van deze thread